Assets/Feature01/Scripts/Input/GamePadCursor.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.InputSystem.Users;
+
+public class GamePadCursor : MonoBehaviour
+{
+    [SerializeField] private GameObject cursor;
+    float speedMultiplier=100f;
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+    public void moveCursorPosition(Vector2 cursorMovementInput,float _cursorSpeed)
+    {
```

Comment:
Why did you need this?
You should have a look at the cursor class on unity API.

Reply:
eta gamepad diye jokhn first e cursor movement  implement korechilam tokhn kora, baad diye dicchi